### DISCUSSION TITLE
Hiding the coordinates?

### DISCUSSION THREAD

**Discussion by yugo76** [Stellarium/stellarium#3205] [Q&A] [answered]
Is there a way to hide the coordinates ? Using a script perhaps?

**Comment by gzotti** [comment]
User Guide, Figure 4.4?

**Reply by yugo76** [reply]
Sorry, I meant the current location. Not the objects being selected with the mouse. The script random_location.ssc has the answer. I just need to figure it out on how to do it without activating the script. A script like "Change current coordinates GUI into 'Unknown'"".

**Reply by alex-w** [reply, marked as answer]
`gui/flag_show_location = false` in `config.ini` file

**Reply by yugo76** [reply]
That did it. Tks a lot!

**Reply by yugo76** [reply]
May I suggest that to be included in the guide/pdf.

**Reply by alex-w** [reply]
See page 365 in the Guide :)

**Comment by 10110111** [comment]
In _Location_ dialog find _Name/City_ field, type the name you want, and then edit the coordinates or altitude. After this the bottom bar should show the name you typed.

**Reply by yugo76** [reply]
This is what I want:
I am studying celestial navigation. I need a way to hide the coordinates GUI. The random location script did it.

![capture1](https://user-images.githubusercontent.com/50803283/234377712-30097a31-dc7b-484a-b7a3-24d82ddf6439.JPG)
![Capture3](https://user-images.githubusercontent.com/50803283/234377736-b1471afd-97db-408e-973a-56d0b6cc7b3b.JPG)

**Reply by yugo76** [reply]
But I need to set up a known location, and have the GUI changed. It's for an exercise for a friend which is learning. I will know the location, but he won't.

**Reply by 10110111** [reply]
My post above that you comment on describes how to achieve exactly what you're asking.

**Reply by yugo76** [reply]
Under [gui] 
flag_show_location                        = false
Got it tks!
